psutil/_pslinux.py

```diff
@@ -785,6 +785,20 @@ def net_if_stats():
 
 # --- disks
 
+def disk_block_stats(device):
+    """Returns a tuple of (ios_in_flight, time_in_queue_ms) for a disk."""
```

Comment:
Please add a reference to https://www.kernel.org/doc/Documentation/block/stat.txt in the docstring.